src/utils.js

```diff
@@ -1,20 +1,12 @@
 'use strict';
 
+var isError = require('iserror');
+
 function isObject(what) {
     return typeof what === 'object' && what !== null;
 }
 
-// Sorta yanked from https://github.com/joyent/node/blob/aa3b4b4/lib/util.js#L560
-// with some tiny modifications
-function isError(what) {
-    var toString = {}.toString.call(what);
-    return isObject(what) &&
```

Comment:
yeah, it's a bit mysterious. Node has moved to a special internal isError that cannot be implemented in plain JS, because objects can override the behaviour of Object.prototype.toString via @@toStringTag

Of course legacy JS could still replace Object.prototype.toString.